stdlib/3/subprocess.pyi

```diff
@@ -6,7 +6,11 @@ from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union,
 from types import TracebackType
 
 _FILE = Union[None, int, IO[Any]]
-_TXT = Union[bytes, Text]
+if sys.version_info >= (3, 6):
+    from builtins import _PathLike
+    _TXT = Union[bytes, Text, _PathLike]
+else:
+    _TXT = Union[bytes, Text]
```

Comment:
I can't comment there, but I noticed this is wrong for the `input` argument to `subprocess.run`, which is currently typed as `Optional[_TXT]` but really only accepts bytes or None.

Reply:
@JelleZijlstra Are you sure?  From looking at https://docs.python.org/3/library/subprocess.html#subprocess.run if `encoding` is specified then `input` must be a string.

```python
import subprocess

subprocess.run(['head', '-n1'], input='Hello World', encoding='ascii')
```

We should fix this type as well, but mind if I do it in a follow up?  I split up Path-like things from Text-like things in this PR.

Reply:
Oh you're right, I didn't think of looking at encoding.